src/connectedk8s/azext_connectedk8s/_help.py

```diff
@@ -26,6 +26,8 @@
       text: az connectedk8s connect -g resourceGroupName -n connectedClusterName --proxy-https https://proxy-url --proxy-http http://proxy-url --proxy-skip-range excludedIP,excludedCIDR,exampleCIDRfollowed,10.0.0.0/24
     - name: Onboard a connected kubernetes cluster by specifying the https proxy, http proxy, no proxy  with cert settings.
       text: az connectedk8s connect -g resourceGroupName -n connectedClusterName --proxy-cert /path/to/crt --proxy-https https://proxy-url --proxy-http http://proxy-url --proxy-skip-range excludedIP,excludedCIDR,exampleCIDRfollowed,10.0.0.0/24
+    - name: Onboard a connected kubernetes cluster with custom onboarding timeout.
+      text: az connectedk8s connect -g resourceGroupName -n connectedClusterName --onboarding-timeout 10
```

Comment:
We decided on parameter introduce for this?

Reply:
If so, should we add it for other commands(update_agents, enable_features, disable_features) which invokes _helm upgrade_ command? For consistency, imo.

Reply:
The rest of the commands do not require an image pull operation(its just config update). The timeout was occuring for ASE due to delay in image pull & container creation